documentation/Installing-Clear-Containers-on-RHEL.md

```diff
@@ -1,206 +1,59 @@
-$ tar xvf gcc-6.2.0.tar.gz
-$ pushd gcc-6.2.0
-$ ./configure --enable-languages=c --disable-multilib --prefix=/usr/local/gcc-6.2.0
-$ make -j5 && sudo make install
-$ popd
-
-```
-
-## Install qemu-lite
-
-Although this package installs the `qemu-lite` version of `qemu`, the binary will still be called `qemu-system-x86_64`.
-Requires at least `gcc` 5.
-
-```
-$ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
-$ export PATH=/usr/local/gcc-6.2.0/bin:$PATH
-$ sudo yum install libcap-ng-devel pixman-devel libattr-devel libcap-devel
-$ curl -L -O https://github.com/01org/qemu-lite/archive/741f430a960b5b67745670e8270db91aeb083c5f.tar.gz
-$ tar xvf 741f430a960b5b67745670e8270db91aeb083c5f.tar.gz
-$ pushd qemu-lite-741f430a960b5b67745670e8270db91aeb083c5f
-$ CC=/usr/local/gcc-6.2.0/bin/gcc ./configure \
-	--disable-bluez         --disable-brlapi \
-	--disable-bzip2         --disable-curl \
-	--disable-curses        --disable-debug-tcg \
-	--disable-fdt           --disable-glusterfs \
-	--disable-gtk           --disable-libiscsi \
-	--disable-libnfs        --disable-libssh2 \
-	--disable-libusb        --disable-linux-aio \
-	--disable-lzo           --disable-opengl \
-	--disable-rbd           --disable-qom-cast-debug \
-	--disable-rdma          --disable-sdl \
-	--disable-seccomp       --disable-slirp \
-	--disable-snappy        --disable-spice \
-	--disable-strip         --disable-tcg-interpreter \
-	--disable-tcmalloc      --disable-tools \
-	--disable-tpm           --disable-usb-redir \
-	--disable-uuid          --disable-vnc \
-	--disable-vte           --disable-vnc-{jpeg,png,sasl} \
-	--disable-xen           --enable-attr \
-	--enable-cap-ng         --enable-kvm \
-	--enable-virtfs         --target-list=x86_64-softmmu \
-	--extra-cflags="-fno-semantic-interposition -O3 -falign-functions=32" \
-	--datadir=/usr/local/share/qemu-lite \
-	--libdir=/usr/local/lib64/qemu-lite \
-	--libexecdir=/usr/local/libexec/qemu-lite
-$ make -j5 && sudo make install
-$ popd
-```
-
-## Install container kernel and rootfs
-
-```
-$ sudo yum-config-manager --add-repo http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/RHEL_7/home:clearlinux:preview:clear-containers-2.1.repo
-$ sudo yum update
-$ sudo yum install linux-container clear-containers-image clear-containers-selinux
-
-```
+For more information on docker installation, see:
+- https://docs.docker.com/engine/installation/linux/rhel/
 
-## Install latest Golang
+## Installation
 
-cc-oci-runtime supports golang 1.7 and 1.8
-The next instructions were taken from [golang site](https://golang.org/doc/install)
+In order to install Clear Containers on RHEL 7.3, just run:
 
 ```
-$ curl -L -O https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz
-$ sudo tar -C /usr/local -xzf go1.8.linux-amd64.tar.gz
-$ export PATH=$PATH:/usr/local/go/bin
-$ mkdir ~/go
-$ export GOPATH=~/go
+$ ./installation/rhel-setup.sh
```

Comment:
I note rhel-setup.sh invokes sudo (as indeed it must for the package installations) - maybe we should add a note here stating that rhel-setup.sh uses sudo, and may ask you for your password (otherwise it might come as some surprise :-).